app/helpers/check_answers_helper.rb

```diff
@@ -1,154 +1,254 @@
 module CheckAnswersHelper
-  SKIPPABLE_QUESTIONS = %w[
-    rooms_number
-    accommodation_cost
-    transport_type
-    transport_cost
-    offer_space_type
-    space_cost
-    offer_care_qualifications
-    care_cost
-    offer_staff_type
-    offer_staff_charge
-    construction_services
-    construction_services_other
-    construction_cost
-    it_services
-    it_services_other
-    it_cost
-  ].freeze
-
-  def items
-    items = questions.map do |question|
-      # We have answers as strings and hashes. The hashes need a little more
-      # work to make them readable.
-
-      next if skip_question?(question)
-
-      next transport_type if question.eql?("transport_type")
-      next offer_care_qualifications if question.eql?("offer_care_qualifications")
-      next offer_staff_type if question.eql?("offer_staff_type")
-      next link_to_parent_page(question, "offer_staff_type") if question.eql?("offer_staff_charge")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_services_other")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_cost")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_services_other")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_cost")
-      next link_to_parent_page(question, "transport_type") if question.eql?("transport_cost")
-      next link_to_parent_page(question, "offer_care_qualifications") if question.eql?("care_cost")
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
-
-      value = concat_answer(session[question], question)
+  def sections
+    %w[
+      accommodation
+      transport
+      space
+      staff
+      care
+      services
+      other_support
+      location
+      business_details
+      contact_details
+    ]
+  end
```

Comment:
I thought there was going to be a comment added to describe the significance of the order.

Reply:
Thanks @reggieb - yes, it's in this commit https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/394/commits/3a2bfbdfa08e2444eac2e0237520aa507b1a547b